storage/zynq7000-flash/flashdrv.c

```diff
@@ -551,13 +551,13 @@ static ssize_t flashdrv_mtdWrite(struct _storage_t *strg, off_t offs, const void
 	ssize_t res = 0;
 	size_t tempSz = 0, chunkSz = 0;
 
-	mutexLock(fdrv_common.regs[strg->dev->id].lock);
+	mutexLock(fdrv_common.regs[strg->dev->ctx->id].lock);
 	chunkSz = offs % strg->dev->mtd->writeBuffsz;
 	if (chunkSz != 0) {
 		chunkSz = len < (strg->dev->mtd->writeBuffsz - chunkSz) ? len : (strg->dev->mtd->writeBuffsz - chunkSz);
-		res = _flashdrv_pageProgram(strg->dev->id, strg->start + offs, data, chunkSz);
+		res = _flashdrv_pageProgram(strg->dev->ctx->id, offs, data, chunkSz);
```

Comment:
Removed `strg->start` without corresponding a change in `_flashdrv_pageProgram`. Is this intended?

Reply:
Nope, it was related to other changes not connected to this PR. Thanks.